fmriprep/workflows/fieldmap/phdiff.py

```diff
@@ -131,6 +132,11 @@ def _pick1st(inlist):
 # Helper functions
 # ------------------------------------------------------
 
+@due.dcite(
+    Doi('10.1006/nimg.2001.1054'),
+    description="Conversion of the input phase-difference map into a fieldmap in Hz",
+    tags=['edu']
+)
```

Comment:
Looks like `Function` doesn't play well with decorated functions. I guess upgrading it to a proper interface would be the easiest solution.